staging/vhost-device-gpu/src/main.rs

```diff
@@ -3,20 +3,21 @@
 //
 // SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
 
-use log::{error, info};
-use std::path::PathBuf;
-use std::process::exit;
-use std::sync::{Arc, RwLock};
-
 use clap::Parser;
+use log::{error, info};
+use std::{
+    path::PathBuf,
+    process::exit,
+    sync::{Arc, RwLock},
+};
 use thiserror::Error as ThisError;
+use vhost_device_gpu::{
+    vhu_gpu::{self, VhostUserGpuBackend},
+    GpuConfig,
+};
 use vhost_user_backend::VhostUserDaemon;
 use vm_memory::{GuestMemoryAtomic, GuestMemoryMmap};
 
-use vhost_device_gpu::vhu_gpu;
-use vhost_device_gpu::vhu_gpu::VhostUserGpuBackend;
-use vhost_device_gpu::GpuConfig;
-
 type Result<T> = std::result::Result<T, Error>;
```

Comment:
The imports in the vhost crates seem to generally be ordered, and include new lines like this:
```Rust
use log::{something};
use std::{something};
// and other std:: imports

use virtio_queue::{DescriptorChain, QueueOwnedT};
// and other libraries ...

use crate::{something}
// and other local imports... 
```

I would say this makes it nicer to see what we are depending on. Interesting how `log` crate is treated special and is in the standard libraries part. I tried to follow this convention, but sometimes I forgot (the IDE adds imports for me).

Reply:
about this, `cargo +nightly fmt` automatically reorders the imports in a better way, basically all i did was grouping and `cargo +nightly fmt`. Going by your suggestion, nightly format reorders them back, so i think it's better to stick to that format.

Reply:
Why `+nightly`? I just use `cargo fmt`, I am not sure if it changes anything though. Are you sure `cargo fmt` reorders the imports? I think if you leave a blank line between some of the groups of imports it doesn't reorder them between the groups (otherwise how else would the other crates do it like this?)

Reply:
yeah right the blank lines, thanks. 
`+nightly fmt` because it allows you to take advantage of the latest features and bug fixes in rustfmt.